chcemvediet/apps/inforequests/admin.py

```diff
@@ -385,20 +389,69 @@ def get_inforequest(self, obj):
         return obj.branch.inforequest
 
     def delete_constraints(self, obj):
-        dependency = []
+        constraints = []
         if obj.type in [Action.TYPES.REQUEST, Action.TYPES.ADVANCED_REQUEST]:
-            dependency.append(format_html(
+            constraints.append(format_html(
                 u'{} is type {}.'.format(admin_obj_format(obj), obj.get_type_display())))
         if len(obj.branch.actions) == 1:
-            dependency.append(format_html(
+            constraints.append(format_html(
                 u'{} is the only action in the branch.'.format(admin_obj_format(obj))))
-        return dependency
+        return constraints
 
     def render_delete_form(self, request, context):
         context[u'delete_constraints'] = self.delete_constraints(context[u'object'])
         context[u'ADMIN_EXTEND_SNOOZE_BY_DAYS'] = ADMIN_EXTEND_SNOOZE_BY_DAYS
         return super(ActionAdmin, self).render_delete_form(request, context)
 
+    @transaction.atomic
+    def delete_selected(self, request, queryset):
```

Comment:
@martinmacko47 Kedze v kode pouzivame `admin.actions.delete_selected`, musime ho povolit na admin-modely, ale zaroven nechceme, aby to bolo k dispozicii adminovi ako samostatna akcia. Preto je nazov funkcie rovnaky.

Reply:
@viliambalaz Mozno by sa zislo zistit, ako built-in `delete_selected` vypnut globalne.

Reply:
:ok:, globalne vypnutie som uskutocnil v `urls.py`,  vychadzal som [odtial](https://stackoverflow.com/questions/63905358/what-is-the-best-practice-best-place-for-enabling-disabling-site-wide-admin-fu)

Reply:
Je `urls.py` vhodne miesto na to? Nebolo by lepsie to spravit v `ready` metode v `ChcemvedietConfig` app configu v `chcemvediet/apps/__init__.py`? To by sa malo zavolat po importe `urls.py` a vsetkych ostatnych veci inicializiucich aplikaciu.

Napriklad tam upravujeme `admin.site` na `AdminSitePlus()` a volame `admin.autodiscover()`.

Reply:
:ok:, vdaka, ano je to urcite lepsie miesto